more_autograding_examples/cpp_count_ts/submissions/buggy.cpp

```diff
@@ -0,0 +1,14 @@
+#include <iostream>
+#include <vector>
+int sum(const std::vector<int> &nums) {
+  int result = 0;
+  for (auto i : nums)
+    result += i;
+  return result;
+}
+
+int main() {
+  std::vector<int> nums = {2, 21, 52, 49, 38, 80};
+  std::cout << "Sum = " << sum(nums) << std::endl;
+  return 0;
+}
```

Comment:
@poorna2152 

Every file in our repositories should have a newline at the end of the last line -- unless there is a specific need for something different for a file.  Many developers use editors that automatically add these missing newlines back in for some/most file types (so if they edit any of these files in the future, this change will be committed and it will be confusing if they didn't change anything else of substance in the file.)  

Github UI is labeling these missing characters in this PR and some of your other PRs too.

Reply:
Thanks for pointing that out. Fixed the issue in this PR. Will fix the issues in other PRs as well